docs/readthedocs/usersguide.md

```diff
@@ -1,85 +1,82 @@
+
+## Requirements
 
 To run and customize the project you need a Java Development Kit (JDK) installation. You can download it `here <https://adoptopenjdk.net/>`_ .
 
 
-Installation
-============
+## Installation
+
 
 You can find and download the source code of the latest stable SIMONA version `here <https://github.com/ie3-institute/simona>`_. Go ahead and clone the repository using git:
 
 .. code-block:: none
 
-   $ git clone https://github.com/ie3-institute/simona.git
+$ git clone https://github.com/ie3-institute/simona.git
 
 
-Running a Standalone Simulation
-===============================
-   
-Running a standalone simulation is a matter of invoking the ``RunSimonaStandalone`` object. You can find it at ``./src/main/scala/edu.ie3/simona/main``. 
-As what you are getting from the git repository is scala source code, you need to compile the code before you can run it. 
-You can either delegate the job to your preferred IDE or build an executable jar manually and use a command line interface (CLI). 
+## Running a Standalone Simulation
 
-Using a CLI
------------
-In order to be able to execute SIMONA from a CLI, you need an executable fat jar. A fat jar contains all compiled classes and dependencies, ready for execution. For building one, you can use a Gradle task of the project. 
 
-#. 
-   Open a CLI and change directories to the top level directory of the project.
+Running a standalone simulation is a matter of invoking the ``RunSimonaStandalone`` object. You can find it at ``./src/main/scala/edu.ie3/simona/main``.
+As what you are getting from the git repository is scala source code, you need to compile the code before you can run it.
+You can either delegate the job to your preferred IDE or build an executable jar manually and use a command line interface (CLI).
+
+### Using a CLI
 
-#. 
-  Execute ``./gradlew shadowJar`` within the CLI. This creates a fat jar of SIMONA inside of the directory ``build/libs``.
+In order to be able to execute SIMONA from a CLI, you need an executable fat jar. A fat jar contains all compiled classes and dependencies, ready for execution. For building one, you can use a Gradle task of the project.
+
+#.
+Open a CLI and change directories to the top level directory of the project.
+
+#.
+Execute ``./gradlew shadowJar`` within the CLI. This creates a fat jar of SIMONA inside of the directory ``build/libs``.
 
 #.
-  For executing a fat jar you need to specify the classpath of the entrypoint of your application. Assuming we are still in the top level directory of our project, the execution command would look as follows:
+For executing a fat jar you need to specify the classpath of the entrypoint of your application. Assuming we are still in the top level directory of our project, the execution command would look as follows:
 
-  .. code-block:: none
+.. code-block:: none
```

Comment:
[Code blocks do not work.](https://simona--200.org.readthedocs.build/en/200/usersguide.html) Please also check below for the same error.

Reply:
Changed the old code-block to the new markdown format.